src/config.h

```diff
@@ -106,7 +106,11 @@
 #define HAVE_ACCEPT4 1
 #endif
 
-#if (defined(__APPLE__) && defined(MAC_OS_10_6_DETECTED)) || defined(__FreeBSD__) || defined(__OpenBSD__) || defined (__NetBSD__)
+#if (defined(__APPLE__) && defined(MAC_OS_10_6_DETECTED)) || \
+    defined(__DragonFly__) || \
```

Comment:
You mentioned it was since a certain version. Can we also add the needed checks to make sure that we don't start failing builds on some obscure version?

Reply:
That is needless, DragonFlyBSD was forked from FreeBSD 4.8, so all versions of DragonFlyBSD should have `kqueue`.

Reply:
Makes sense